src/Microsoft.AspNetCore.Server.HttpSys/AsyncAcceptContext.cs

```diff
@@ -179,6 +175,37 @@ internal uint QueueBeginGetContext()
             return statusCode;
         }
 
+        // Take this out to a static method that returns a native request context. 
+        internal static NativeRequestContext AllocateNativeRequest(AsyncAcceptContext acceptResult, uint? size = null, ulong requestId = 0)
```

Comment:
remove static and return parameter, update the member directly. Also remove acceptResult parameter.